database/src/main/resources/db/migration/V88__fradrag_i_beregning_til_grunnlag.sql

```diff
@@ -0,0 +1,29 @@
+with fradrag_fra_revurdering (behandlingId, opprettet, fradrag) as (
+    select id, opprettet, jsonb_array_elements(beregning -> 'fradrag')
+    from revurdering
+),
+     mapped_values as (select behandlingId as                                      behandlingId,
+                              opprettet    as                                      opprettet,
+                              (fradrag -> 'periode' ->> 'fraOgMed')::date          fraogmed,
+                              (fradrag -> 'periode' ->> 'tilOgMed')::date          tilogmed,
+                              fradrag -> 'tilhører'                                tilhører,
+                              fradrag -> 'fradragstype'                            fradragstype,
+                              round((fradrag ->> 'månedsbeløp')::double precision) månedsbeløp,
```

Comment:
Kan jo sjekke om det faktisk eksisterer noen som ikke har verdi xx.0 i basen - hvis ikke bør det vel være trygt å jazze det over til int.

Reply:
Finnes en del som faktiskt er doubles ja ☺️ vet ikke hvordan vi bør hantera det då. Antingen så runder vi av eller så gør vi om månedsbeløp till double fra int. Borde vara mest riktig att gøra om tabellen isåfall?